daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -97,27 +98,29 @@ cmdTest numProcessors =
     <> fullDesc
   where
     cmd = runTestsInProjectOrFiles
-      <$> many inputFileOpt
+      <$> projectRootOpt
```

Comment:
It probably makes sense to group the `projectRootOpt` and `projectCheckOpt` into a single options parser, returning a type of "project options", and have a damlc wrapper over `withProjectRoot` that handles these options specifically.

Reply:
Good idea, done.